OktaJWT/ThirdParty/JSONWebToken/TokenValidator.swift

```diff
@@ -40,7 +40,7 @@ public func &(lhs : JSONWebTokenValidatorType, rhs : JSONWebTokenValidatorType)
             } else {
                 return nil
             }
-            }.flatMap {$0}
+            }.compactMap {$0}
```

Comment:
For changes to Third Party libraries, we need to update the file's header to include:

```diff
  //
  //  TokenValidator.swift
  //
  //  Created by Antoine Palazzolo on 23/11/15.
+ //  Modifications Copyright 2018 Okta, Inc.
  //
```